sources/reference/resource-state.md

```diff
@@ -0,0 +1,38 @@
+page_main_title: state
+main_section: Reference
+sub_section: Resources
+
+# state
+A `state` resource type is used to create a central state that can be used across the entire pipelines. This resource can be used as an IN input and OUT output for [any job](jobs-overview/). The timezone used for triggering jobs is UTC.
```

Comment:
i would say "can be used as an IN and OUT step" rather than "IN input" "OUT output"